android/app/src/main/AndroidManifest.xml

```diff
@@ -3,18 +3,20 @@
     android:versionCode="1"
     android:versionName="1.0">
 
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" /><!-- WebRTC -->
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
I'm not sure why it was necessary to remove the `WebRTC` comment. I know I added it because the app crashed for me and it wasn't obvious that I needed the permission.